src/Arcus.Testing.Assert/AssertCsv.cs

```diff
@@ -623,22 +619,77 @@ internal static CsvTable Load(string csv, AssertCsvOptions options)
             return new CsvTable(headerNames, rows, csv, options);
         }
 
-        private static void EnsureAllRowsSameLength(string csv, string[][] rawRows)
+        private static string[][] SplitCsv(string csv, AssertCsvOptions options)
```

Comment:
Wanted to keep this implementation as simple as possible, feel free to play or propose something else. Didn't want to use regex's here, as it might cause some overhead with bigger CSV files.

Reply:
Looks fine to me. Simple is good sometimes. We'll see during testing or in the future if we need to change that.

Reply:
Yeah, I believe due to my lack of using a lot of CSV files in projects, there might be some edge cases it's missing; but for regular use, it might be enough.